test/setup/index.ts

```diff
@@ -133,7 +133,7 @@ export async function deployMonitor(): Promise<string> {
     await predeploy(integrationId);
 
     // TODO: hack, rewrite this
-    const imagePullPolicy = testPlatform === 'kind' ? 'Never' : 'Always';
+    const imagePullPolicy = testPlatform === 'kind' || testPlatform === 'openshift3' ? 'Never' : 'Always';
```

Comment:
A small comment. Just to avoid the code gets complicated. What do you think if we create a function, per platform, that specify which policy we should use to image pull?
Not sure if it will help us, or make the code harder to understand.

Reply:
I think at this point I'd leave it here as I it would require extending the whole interface so we can do that once it does get complicated, wdyt?